### DISCUSSION TITLE
Changes in release v23.02.29

### DISCUSSION THREAD

**Discussion by Gyosa3** [huizebruin/s0tool#117] [General] [answered]
Hi, I am running your box on my water meter and it's great :)

I was taken by surprise with the latest version, as I clicked "update" without checking the release note. I digested the change to "watermeter flow", that's a better name, but I seem to have missed what happened with the pulse sensor.

"wijziging van pulse_counter naar pulse_meter"

Am I right if I understand that you have completely removed it both in the web page and in the HA device/entities?  Just to be sure my setup is not going wrong...

Also, I received your setup with an initial meterstand of about 8m3, while my water meter was new, and I have not found a way to change the value to match my meter. In your docs it says that you can change the value in "Developer Tools" and set the state of that sensor, but that does not update the value in the device, probably because a sensor is read-only in HA?

Do you have any other method to write the value in the board itself? Shall I crate a new issue in Github for this maybe?

Thanks!!

**Comment by huizebruin** [comment, marked as answer]
Hello @Gyosa3 

pulse sensor is going to the background, i had a lot of questions about it, why is off but i see the counting.

its strange u cant change the meterstand from the initial stand, that's new for me.

i wil go looking what's that about it why you cant change it anymore .

and if you would make a issue for the meterstand , please so i have reminder to look at that.

**Comment by huizebruin** [comment]
how its working these days?
